spec/source/conventions.html

```diff
@@ -0,0 +1,107 @@
+<emu-clause id="conventions">
+  <h1>Notational Conventions</h1>
+
+  <p>
+    This standard uses a subset of the notational conventions of the ECMAScript 2015 Language Specification (ECMA-262 6th Edition), as ES2015:
+  </p>
+
+  <ul>
+    <li>Object Internal Methods and Internal Slots, as described in ES2015, 6.1.7.2.</li>
+    <li>Algorithm conventions, including the use of abstract operations, as described in ES2015, 7.1, 7.2, 7.3.</li>
+    <li>Internal Slots, as described in ES2015, 9.1.</li>
+    <li>The List and Record Specification Type, as described in ES2015, 6.2.1.</li>
+  </ul>
+
+  <emu-note>
+    <p>
+      As described in the ECMAScript Language Specification, algorithms are used to precisely specify the required semantics of ECMAScript constructs, but are not intended to imply the use of any specific implementation technique. Internal slots are used to define the semantics of object values, but are not part of the API. They are defined purely for expository purposes. An implementation of the API must behave as if it produced and operated upon internal slots in the manner described here.
+    </p>
+  </emu-note>
+
+  <p>
+    As an extension to the Record Specification Type, the notation "[[name]]" denotes a field whose name is given by the variable name, which must have a String value. For example, if a variable s has the value "a", then [[s]] denotes the field [[a]].
+  </p>
+
+  <p>
+    For ECMAScript objects, this standard may use variable-named internal slots: The notation "[[name]]" denotes an internal slot whose name is given by the variable name, which must have a String value. For example, if a variable s has the value "a", then [[s]] denotes the [[a]] internal slot.
+  </p>
+
+  <emu-clause id="sec-well-known-intrinsic-objects">
+    <h1>Well-Known Intrinsic Objects</h1>
+
+    <p>
+      The following table extends the Well-Known Intrinsic Objects table defined in ES2015, 6.1.7.4.
+    </p>
+
+    <h2 id="table-7">Table 7 — Well-known Intrinsic Objects (Extensions)</h2>
+    <table>
+      <thead>
+        <tr>
+          <th>Intrinsic Name</th>
```

Comment:
In the loader spec we reference this as "Internal Slot" and entries in that column will be reference as `[[Something]]`. Can we agree on the right way to define this? /cc @bterlson